PWGMM/Tasks/multiplicityPbPb.cxx

```diff
@@ -115,6 +118,19 @@ struct multiplicityPbPb {
 
     histos.fill(HIST("Anton/NtrkZvtxEvents"), trackCounter, collision.posZ());
   }
+  PROCESS_SWITCH(multiplicityPbPb, process, "process real and MC data", true);
```

Comment:
The function with the name `process()` is executed unconditionally, there is no need to add a switch. If you want to make the function switchable, it should be named differently.